Makefile

```diff
@@ -1,5 +1,8 @@
+PREFIX ?= /usr/local
+LIBDIR ?= /lib
+
 clientbuffer.so : clientbuffer.cpp
 	znc-buildmod clientbuffer.cpp
 
 install: clientbuffer.so
-	install clientbuffer.so /usr/lib/znc/
+	install -Dm 644 clientbuffer.so $(DESTDIR)$(PREFIX)$(LIBDIR)/znc/clientbuffer.so
```

Comment:
I think .so files should be +x.

What are you packaging for?

Reply:
Arch Linux

Yes, you're right, they should be 755, sorry.